kubernetes-py-jenkins/README.md

```diff
@@ -0,0 +1,130 @@
+```
+
+Create a new stack:
+
+```bash
+    $ pulumi stack init dev
+```
+
+Create configuration keys for the root username and password for the Jenkins instance we are
+about to create:
+
+```bash
+    $ pulumi config set username <your desired username>
+    $ pulumi config set password <your desired password> --secret
+```
+
+Configure Kubernetes to run without Minikube:
+
+```bash
+    $ pulumi config set kubernetes-py-jenkins:isMinikube 0
+```
+
+Preview the deployment of the application:
+
+```bash
+    $ pulumi preview
+    Previewing update of stack 'dev'
+        Type    Name    Status        Info
+    *   global  global  unchanged
+    Previewing changes:
+
+        Type                                            Name                                             Status        Info
+    *   global                                          global                                           no change
+    +   pulumi:pulumi:Stack                             kubernetes-py-jenkins-dev                        create
+    +   └─ jenkins:jenkins:Instance                     dev                                              create
+    +      ├─ kubernetes:core:v1:Secret                 dev-secret                                       create
+    +      ├─ kubernetes:core:v1:PersistentVolumeClaim  dev-pvc                                          create
+    +      ├─ kubernetes:core:v1:Service                dev-service                                      create
+    +      └─ kubernetes:extensions:v1beta1:Deployment  dev-deploy                                       create
+
+    info: 6 changes previewed:
+        + 6 resources to create
+```
+
+Perform the deployment:
+
+```bash
+    $ pulumi up --skip-preview
+    Updating stack 'dev'
+        Type    Name    Status        Info
+    *   global  global  unchanged
+    Performing changes:
+
+        Type                                            Name                                             Status        Info
+    *   global                                          global                                           unchanged
+    +   pulumi:pulumi:Stack                             kubernetes-py-jenkins-dev                        created
+    +   └─ jenkins:jenkins:Instance                     dev                                              created
+    +      ├─ kubernetes:core:v1:Secret                 dev-secret                                       created
+    +      ├─ kubernetes:core:v1:PersistentVolumeClaim  dev-pvc                                          created
+    +      ├─ kubernetes:core:v1:Service                dev-service                                      created
+    +      └─ kubernetes:extensions:v1beta1:Deployment  dev-deploy                                       created
+
+    info: 6 changes performed:
+        + 6 resources created
+    Update duration: 2m30.397621136s
+```
+
+The deployment is complete! Use `kubectl` to see the Service that we just deployed:
```

Comment:
Rather than looking up the IP with `kubectl`, you could export the value from the Service status. I realize this is a port of an existing example, so if you'd prefer to leave as is, can you open an issue to update these two examples?

Reply:
Thanks. I'll try to open an issue for the two examples.

Reply:
Created issue [#718](https://github.com/pulumi/examples/issues/718). Will follow up on it and fix it in a separate pull request.